src/components/packages/Package.js

```diff
@@ -129,7 +129,7 @@ class Package extends Component {
 
         return <div className="fs-selected-pricing-license-quantity">
             {selectedPricing.sitesLabel()}
-            { ! planPackage.is_free_plan && <Tooltip>
+            { ! planPackage.is_free_plan && <Tooltip direction={this.getTooltipDirection()}>
                 <Fragment>
                     If you are running a multi-site network, each site in the network requires a license.{pricingLicenses.length > 0 ? 'Therefore, if you need to use it on multiple sites, check out our multi-site prices.' : ''}
```

Comment:
We need to add a space after `license.(space here){pricingLicenses.length > 0 ? ...}`.

I understand the code was already there, but since we've spotted this, let us fix this 👍